.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules
+.env
```

Comment:
You definitely added this after you had committed .env. Once you have a file committed, ignore won't remove it. You have to run `git rm` to get it out of the repo and then you can .gitignore it.